console/console-init/ui/src/modules/iot-certificates/components/IoTCertificate/IoTCertificate.tsx

```diff
@@ -9,33 +9,43 @@ import {
   CertificateForm,
   CertificateCard
 } from "modules/iot-certificates";
+import { PageSection } from "@patternfly/react-core";
 
 export interface IIoTCertificateProps {
   certificate: IIoTCertificate;
+  onEdit: (certificate: IIoTCertificate) => void;
```

Comment:
ICertificateCardProps have already all these properties so don't need to declare again, just import ICertificateCardProps interface and extends with IIoTCertificateProps interface. If you need specific properties and want to remove some properties, use Omit or Pick based on need.

export interface IIoTCertificateProps extends  ICertificateCardProps{
   id:string
  certificate: IIoTCertificate;
}